ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/.content.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@DeepaliAgrwl  Why this file is part of logo comp?

Reply:
reverted back this file